packages/terra-progress-bar/src/terra-dev-site/doc/progress-bar/ProgressBar.1.doc.mdx

```diff
@@ -0,0 +1,57 @@
+import { Badge } from 'terra-progress-bar/package.json?dev-site-package';
+
+import ProgressBarDefault from '../example/ProgressBarDefault?dev-site-example';
+import ProgressBarSize from '../example/ProgressBarSize?dev-site-example';
+import ProgressBarColor from '../example/ProgressBarColor?dev-site-example';
+import ProgressBarTwoColors from '../example/ProgressBarTwoColors?dev-site-example';
+import ProgressBarCustomizedColors from '../example/ProgressBarCustomizedColors?dev-site-example';
+
+import ProgressBarPropsTable from 'terra-progress-bar/src/ProgressBar?dev-site-props-table';
+
+<Badge />
+# Terra Progress Bar
```

Comment:
This is not being displayed correctly on the dev site. I think a new line is required before this.